services/channel/service/src/index.ts

```diff
@@ -0,0 +1,166 @@
+import {
+  createPostgresPoolConnectivityMetric,
+  setupLoginPgPool,
+  setupOwnerPgPool,
+} from '@axinom/mosaic-db-common';
+import {
+  AuthenticationConfig,
+  IdGuardErrors,
+  setupManagementAuthentication,
+  setupManagementGQLSubscriptionAuthentication,
+} from '@axinom/mosaic-id-guard';
+import { getServiceAccountToken } from '@axinom/mosaic-id-link-be';
+import { createRabbitMQConnectivityMetric } from '@axinom/mosaic-message-bus';
+import {
+  closeHttpServer,
+  handleGlobalErrors,
+  isServiceAvailable,
+  Logger,
+  MosaicError,
+  MosaicErrors,
+  mosaicFavicon,
+  setupGlobalConsoleOverride,
+  setupGlobalLogMiddleware,
+  setupGlobalSkipMaskMiddleware,
+  setupHttpServerWithWebsockets,
+  setupLivenessAndReadiness,
+  setupMonitoring,
+  setupServiceHealthEndpoint,
+  setupShutdownActions,
+  tenantEnvironmentIdsLogMiddleware,
+  trimErrorsSkipMaskMiddleware,
+} from '@axinom/mosaic-service-common';
+import { createMessagingMetric } from '@axinom/mosaic-transactional-inbox-outbox';
+import express from 'express';
+import helmet from 'helmet';
+import { PoolConfig } from 'pg';
+import {
+  applyMigrations,
+  getFullConfig,
+  HEALTH_CHECK_ROUTING_KEY,
+  setIsLocalizationEnabledDbFunction,
+} from './common';
+import { registerTypes, syncPermissions } from './domains';
+import { setupPostGraphile } from './graphql';
+import { registerMessaging } from './messaging';
+
+const logger = new Logger({ context: bootstrap.name });
+
+async function bootstrap(): Promise<void> {
+  handleGlobalErrors(logger);
+  setupGlobalSkipMaskMiddleware(trimErrorsSkipMaskMiddleware);
+  setupGlobalConsoleOverride(logger);
+  const app = express();
+  const config = getFullConfig();
+  setupGlobalLogMiddleware([tenantEnvironmentIdsLogMiddleware(config)]);
+
+  // Create a HTTP server with all WebSocket middleware registered, needed for GQL subscriptions.
+  const authConfig: AuthenticationConfig = {
+    tenantId: config.tenantId,
+    environmentId: config.environmentId,
+    authEndpoint: config.idServiceAuthBaseUrl,
+  };
+  const httpServer = setupHttpServerWithWebsockets(
+    app,
+    logger,
+    setupManagementGQLSubscriptionAuthentication(authConfig),
+  );
+  const { readiness } = setupLivenessAndReadiness(config);
+
+  // Check ID service is available
+  if (!(await isServiceAvailable(config.idServiceAuthBaseUrl))) {
+    throw new MosaicError(IdGuardErrors.IdentityServiceNotAccessible);
+  }
+
```

Comment:
I think somewhere around here, to align with `media-service` implementation, we should have an `await updateConfigWithActualLocalizationAvailability(config, logger);`